hlint/hlint.dhall

```diff
@@ -564,6 +564,28 @@ in [ rule.Arguments { arguments =
    , warnReexport "getCompose" "Data.Functor.Compose"
    , warnReexport "Identity"    "Data.Functor.Identity"
    , warnReexport "runIdentity" "Data.Functor.Identity"
+   , warnReexport "Contravariant"          "Data.Functor.Contravariant"
+   , warnReexport "contramap"              "Data.Functor.Contravariant"
+   , warnReexportOp ">$"                   "Data.Functor.Contravariant"
+   , warnReexport "phantom"                "Data.Functor.Contravariant"
+   , warnReexportOp ">$<"                  "Data.Functor.Contravariant"
+   , warnReexportOp ">$$<"                 "Data.Functor.Contravariant"
+   , warnReexportOp "$<"                   "Data.Functor.Contravariant"
+   , warnReexport "Predicate"              "Data.Functor.Contravariant"
+   , warnReexport "getPredicate"           "Data.Functor.Contravariant"
+   , warnReexport "Comparison"             "Data.Functor.Contravariant"
+   , warnReexport "getComparison"          "Data.Functor.Contravariant"
+   , warnReexport "defaultComparison"      "Data.Functor.Contravariant"
+   , warnReexport "Equivalence"            "Data.Functor.Contravariant"
+   , warnReexport "getEquivalence"         "Data.Functor.Contravariant"
+   , warnReexport "defaultEquivalence"     "Data.Functor.Contravariant"
+   , warnReexport "comparisonEquivalence"  "Data.Functor.Contravariant"
+   , warnReexport "Op"                     "Data.Functor.Contravariant"
+   , warnReexport "getOp"                  "Data.Functor.Contravariant"
```

Comment:
Hmm 🤔 
These rules are only valid for the GHC >= 8.6, but the `.hlint.yaml` is not that smart to detect it. So if users will import this using GHC-8.4 in their project they still will receive these warnings (the module in the `contravariants` package has the same name), but actually, they shouldn't, as `relude` is not exporting these functional for earlier GHC versions.

Reply:
Hmm.. I didn't think about it earlier but I agree. We don't want to give warnings when it's not possible. It's probably okay if the functions are not only in GHC-8.0.2, but GHC 8.6.1 is quite new... These rules are valuable, so we can just comment them for now.

Reply:
Okay I understood your reasoning here. Makes sense. I'll comment out all the rules.